doc/gallery-src/analysis/run_blockMcnpMaterialCard.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 """
 Write MCNP Material Cards
-=========================
+=========================.
```

Comment:
This is an error in the auto fix

Reply:
I agree, I will have to fix these.  Sorry!